coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Products/ProductResponse.cs

```diff
@@ -84,5 +89,13 @@ public class ProductResponse
         /// <example> Manager, Board </example>
         [Required]
         public IEnumerable<UserGroup> AllowedUserGroups { get; set; } = new List<UserGroup>();
+
+        /// <summary>
+        /// The menu items that this product can be used on.
+        /// </summary>
+        /// <value>Menu items</value>
+        /// <example>Cappuccino, Caffe Latte</example>
+        /// <remarks>Optional for backwards compatibility</remarks>
+        public IEnumerable<MenuItemResponse> MenuItems { get; set; } = new List<MenuItemResponse>();
```

Comment:
Suggest to rename to EligibleMenuItems to clearly express the intent

Reply:
Addressed in fe39ffed2d7d416aee95f353586cf80e70afeb6f